.eslintignore

```diff
@@ -8,8 +8,8 @@ bower_components
 /src/fixtures/vislib/mock_data
 /src/ui/public/angular-bootstrap
 /src/ui/public/flot-charts
-/src/ui/public/kuery/ast/kuery.js
-/src/ui/public/kuery/ast/legacy_kuery.js
+/src/utils/kuery/ast/kuery.js
+/src/utils/kuery/ast/legacy_kuery.js
```

Comment:
Out of interest: why are we ignoring those two files for linting? :)

Reply:
They're generated by [PEG](https://pegjs.org/)